tests/E2E/test/campaigns/common_scenarios/customer.js

```diff
@@ -113,10 +113,10 @@ module.exports = {
       test('should select the searched client', () => client.waitForExistAndClick(Customer.select_customer));
       test('should click on the "Bulk actions" button', () => client.waitForExistAndClick(Customer.bulk_actions_button));
       test('should click on the "Delete selected" button', () => client.waitForExistAndClick(Customer.bulk_actions_delete_button));
-      test('should accept the currently displayed alert dialog', () => client.alertAccept());
-      test('should choose the option that allows customers to register again with the same email address', () => client.waitForExistAndClick(Customer.delete_first_option));
+      //test('should accept the currently displayed alert dialog', () => client.alertAccept());
```

Comment:
If you don't need it anymore, please remove it.

Reply:
100% correct, and done.